app/models/decorators/mvi_user_decorator.rb

```diff
@@ -12,6 +14,23 @@ def create
     raise Common::Exceptions::ValidationErrors, @user unless @user.valid?
     message = create_message
     response = @mvi_service.find_candidate(message)
+    user_with_mvi_attributes(response)
+  rescue MVI::RecordNotFound
+    # TODO(AJD): add metric
+    Rails.logger.error "MVI record not found for user: #{@user.uuid}"
+    @user.attributes = { mvi: NOT_FOUND }
```

Comment:
Gotcha.  For develops who don't know this nuance, it would be nice to make this more readable.  Would:
```ruby
@user.mvi = NOT_FOUND
```
accomplish the same thing?